content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+
+![Resource Allocation](/images/gnome-boxes/boxes-create-4.png)
+
+At this point, you should see the Resource Allocation screen where you can select the amount of RAM and the disk size for the VM. The amount of RAM and drive space you give to the VM will be dependant on the hardware requirements for the OS you are looking to install.
+
+If you are unsure what values to use, 8GB of RAM and 60GB for the maximum disk size will be enough for most operating systems to install, with some room for growth as you use the VM.
+
+### Performing the OS Install
+
+![Install the VM](/images/gnome-boxes/boxes-vm-install.png)
+
+After selecting the resources for your VM, the VM will boot the selected operating system installer. Likely, you will see some odd graphical behavior, such as the black bars shown in the above screenshot. This is because some of the drivers needed for the VM are not installed yet. These oddities can be ignored for now, and we will install the drivers later.
+
+The install process for your chosen operating system will be just like any other install, with nothing VM specific at this time. Proceed through the OS install using the OS's guides, or possibly one of the System76 guides linked below:
+
+#### Linux
+
+Install Pop!_OS - [https://support.system76.com/articles/install-pop/](https://support.system76.com/articles/install-pop/)
+
+Install Ubuntu - [https://support.system76.com/articles/install-ubuntu/](https://support.system76.com/articles/install-ubuntu/)
+
+#### Windows
+
+![Windows Express Install](/images/gnome-boxes/boxes-windows-expressinst.png)
+
+When you attempt to create a Windows VM, you will likely see a prompt for the "Express Install" feature of GNOME Boxes. You can certainly use the Express Install with your Windows product key. However, for this article turn off the Express Install, then click "Next" at the top right to move to the standard installer set up.
+
+Install Windows - [https://support.system76.com/articles/windows/](https://support.system76.com/articles/windows/)
+
+### Install GNOME Boxes Drivers
+
+GNOME Boxes has some features that require drivers to be installed into the Guest operating system for better performance and to better integrate the VM with the VM software.
+
+#### Linux Guests
+
+There are 2 drivers needed to get the full functionality out of your Linux VM with GNOME Boxes. The first is the `spice-vdagent` driver, and the other is the `spice-webdavd` driver. Both drivers are provided by the Open Source [Spice Project](https://www.spice-space.org/).
+
+Some Linux distributions include one or both of these drivers out of the box, while others will need them both installed. Additionally, your Linux distribution may not use the names used above, so you may need to reach out to those maintainers for more information. You can see an example of this discrepency below in the Arch Linux section. On Arch, the `spice-webdavd` package does not exist, but the `phodav` package works as a replacement.
```

Comment:
Minor clarity - This statement may be confusing since line 106 states the drivers are required. I would suggest combining these paragraphs: 

"Depending on the distribution, you may need to install additional drivers to gain video compatibility and file-sharing features. The [Spice Project](https://www.spice-space.org/) provides the following drivers: 

**spice-webdavd** - This driver allows sharing folders and files between the Host and VM.

**spice-vdagent**

- Allows the VM screen to resize automatically to match the GNOME boxes window
- Provides ability to copy/paste between the Host and VM
- Supports transferring files from client to agent (according to the [README.md])(https://gitlab.freedesktop.org/spice/linux/vd_agent)

>Note: The driver names may vary by distribution, as shown in the below Arch example: `phodav` is used instead of the `spice-webdavd` package name. We recommend referencing your specific distributions documentation for equivalent or alternative driver names.

Reply:
Really like the look of this rewrite! This should resolve the above 2 comments as well. I'll get this implemented when I add the new photos.